gatling-maven-plugin/src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -223,6 +229,35 @@ public void execute() throws MojoExecutionException {
     }
   }
 
+  private void iterateBySimulations(Toolchain toolchain, List<String> jvmArgs, List<String> testClasspath, Collection<String> simulations) throws Exception {
+    Exception exc = null;
+    String[] simulationsArray = (String[]) simulations.toArray();
```

Comment:
It's not List, it's Collection and there are no get method, that's why I asked you why do you use this interface instead of List